documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -207,7 +207,10 @@ The source configuration uses the same configuration properties that are describ
 |[[debezium-source-offset-storage]]<<debezium-source-offset-storage, `debezium.source.offset.storage`>>
 |`org.apache.kafka.connect.storage.FileOffsetBackingStore`
 |Class to use for storing and retrieving offsets for non-Kafka deployments.
-To use Redis to store offsets, use `io.debezium.storage.redis.offset.RedisOffsetBackingStore`
+Available options
+
+* `org.apache.kafka.connect.storage.FileOffsetBackingStore` for non-Kafka deployments  
+* `io.debezium.storage.redis.offset.RedisOffsetBackingStore` for Redis deployments
```

Comment:
I think we can also add `MemoryOffsetBackingStore` and `JdbcOffsetBackingStore` here as well but this is up for discussion.

Reply:
I have no objection, if you want to add that to the docs as well, :+1:

Reply:
@Naros I added those two and amended the commit. Thank you.